### ISSUE TITLE
Invert value of or remove percentage display header

### ISSUE THREAD

**Issue by fl380** [konnectedvn/lovelace-vertical-slider-cover-card#17]
**Is your feature request related to a problem? Please describe.**
Not specifically related to problem but a request for additional functionality

**Describe the solution you'd like**
Could you please provide or suggest an option to either invert or remove the percentage header so that 0% is at the top of the slider (currently 100%) and 100% is at the bottom (currently 0%) 

**Describe alternatives you've considered**
If this is not feasible or too time consuming then an option to not display the percentage header please

**Additional context**
I've started studying and understanding the code with my limited coding experience. I'm happy to work on it myself to submit a pull request but any tips or suggestions will be appreciated please.
The slider moves in the correct direction and sets my blinds to a matching position, only the percentage header is displayed incorrectly.

thank you

**Comment by wolverinevn** [comment]
Could you embed a screenshot of incorrect header?

**Comment by Wampukas** [comment]
> **Is your feature request related to a problem? Please describe.**
> Not specifically related to problem but a request for additional functionality
> 
> **Describe the solution you'd like**
> Could you please provide or suggest an option to either invert or remove the percentage header so that 0% is at the top of the slider (currently 100%) and 100% is at the bottom (currently 0%)
> 
> **Describe alternatives you've considered**
> If this is not feasible or too time consuming then an option to not display the percentage header please
> 
> **Additional context**
> I've started studying and understanding the code with my limited coding experience. I'm happy to work on it myself to submit a pull request but any tips or suggestions will be appreciated please.
> The slider moves in the correct direction and sets my blinds to a matching position, only the percentage header is displayed incorrectly.
> 
> thank you.  



@fl380 I agree with you. @wolverinevn  My both shades are open and it showned as closed. It would be ok if it will be inverse option. https://prnt.sc/10h6maw

**Comment by Taraman17** [comment]
![grafik](https://user-images.githubusercontent.com/4706817/150079587-cf35421a-bdee-49e0-b129-06b33e2e563c.png)
I have the same problem and here is the screenshot.

**Comment by Taraman17** [comment]
I made a proposal, that would implement removal of the %-value:
See #23

**Comment by Taraman17** [comment]
Something went wrong with the commit of the changes.
See https://github.com/konnectedvn/lovelace-vertical-slider-cover-card/commit/82d7fab68de37d5823a7ff24ff1cde9d42352242

**Comment by Frank-Bemelman** [comment]
I am also trying to hide the name and position, but adding showName: false in the card yaml does not have any effect. Same with showPosition:false. Not sure what goes wrong here...
